nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -541,7 +541,7 @@ in
 
     # Adapt from https://gitlab.gnome.org/GNOME/gnome-build-meta/-/blob/3.38.0/elements/core/meta-gnome-core-developer-tools.bst
     (mkIf serviceCfg.core-developer-tools.enable {
-      environment.systemPackages = (with pkgs.gnome; removePackagesByName [
+      environment.systemPackages = (with pkgs.gnome; utils.removePackagesByName [
```

Comment:
With the `with`, it might be nicer to use `lib.utils` to make it more explicit where it comes from.